tools/image_processing/imagej2/imagej2_watershed_binary/imagej2_watershed_binary.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<tool id="imagej2_watershed_binary" name="Watershed segmentation" version="1.0.0">
+    <description>of binary image with ImageJ2</description>
+    <macros>
+        <import>imagej2_macros.xml</import>
+    </macros>
+    <expand macro="fiji_20141125_requirements" />
```

Comment:
@bgruening Can you clarify your advice here:

expand macro="fiji_20141125_requirements"
General comment: I think this should be renamed to something without a version number. This would have the advantage that we don't need to change every tool if we increase the dependency.

If we use a generic requirements tag, how can we ensure that tools can get access to this specific requirement combination (requirement type="package" version="20141125">fiji) when future fiji versions are released?

Is it safe to assume that we will always be able to update the fiji dependency with the latest fiji release and all tools will produce the same results that they did in the past?  If so, ok - perhaps I'm not thinking clearly on this... ;)